### PR TITLE
Fix Avalanche client desc - typos

### PR DESCRIPTION
**Pull request by conor2e**
## Description

- Added correct description of the Avalanche client used for elastic archive nodes.
- Added a missing link and modified some punctuation/typos.
- Reordered Cronos related paragraph.

### CODE REVIEW

**Review by ginstagram — DISMISSED** [review]
LGTM

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.5.14+{BUILD_NUM}
+3.5.15+{BUILD_NUM}
```
Comment (akegaviar) [inline VERSION:1]: @PekopT is this proper versioning?

Comment (conor2e) [reply VERSION:1]: My understanding is that the build version number now needs to change in every PR, so wont actually represent a release version. @cthtuf, can you confirm?

Comment (cthtuf) [reply VERSION:1]: It's proper. First two digits should be the same as in ui, third one should be incremented every PR

**Review by akegaviar — APPROVED** [review]

**Review by ginstagram — APPROVED** [review]
LGTM

**Pull request merged**